cmd/yesiscan/main.go

```diff
@@ -59,16 +63,30 @@ var program string
 //go:embed .version
 var version string
 
+// autoConfigURI is set via -ldflags build time flags.
+var autoConfigURI string
```

Comment:
ldflags is linker flags. You might want to ask the golang community if you think there is an improvement here. This has been part of golang since the beginning.

Reply:
I wasn't interested in changing anyone's mind, I was interested in learning why it makes sense, if you happen to know. If you don't know why, that's fine. Maybe I can find someone who does.